features/facebookImport/src/views/ministories/story.jsx

```diff
@@ -15,6 +16,14 @@ class Story extends Component {
         return this._analyses;
     }
 
+    get mode() {
+        return this._mode;
+    }
+
+    set mode(mode) {
+        this._mode = Story.MODES[mode] || this.mode;
```

Comment:
this.mode is the same as this._mode. Don't know if one is better than the other, here. You might want to add some additional logic in `mode`, I guess.

Reply:
At any rate, if the modes are fixed and known in advance, having `setThisMode` and `setThatMode` would probably be a safer way of doing it.